packages/algob/src/internal/deployer.ts

```diff
@@ -489,7 +498,87 @@ class DeployerBasicMode {
 	async getReceiptTxns(txns: Transaction[]): Promise<TxnReceipt[]> {
 		return await this.algoOp.getReceiptTxns(txns);
 	}
+
+	/**
+	 * Creates an algosdk.Transaction object based on execParams and suggestedParams
+	 * @param execParams execParams containing all txn info
+	 * @param txParams suggestedParams object
+	 * @returns array of algosdk.Transaction objects
+	 */
+	makeTx(execParams: wtypes.ExecParams[], txParams: algosdk.SuggestedParams): Transaction[] {
+		const txns: Transaction[] = [];
+		for (const [_, txn] of execParams.entries()) {
+			txns.push(mkTransaction(txn, txParams));
+		}
+		return txns;
+	}
+
+	/**
+	 * Signes a Transaction object with the signer
+	 * @param transaction transaction object.
+	 * @param signer object that signes the transaction
+	 * @returns SignedTransaction
+	 */
+	async signTx(
+		transaction: algosdk.Transaction,
+		signer: wtypes.Sign
+	): Promise<SignedTransaction> {
+		let decodedResult: Uint8Array;
+		switch (signer.sign) {
+			case wtypes.SignType.SecretKey: {
+				decodedResult = transaction.signTxn(signer.fromAccount.sk);
+				break;
+			}
+			case wtypes.SignType.LogicSignature: {
+				signer.lsig.lsig.args = signer.args ?? [];
+				decodedResult = algosdk.signLogicSigTransactionObject(transaction, signer.lsig).blob;
+				break;
+			}
+			default: {
```

Comment:
We now also have MultiSignature. Can we also add for that as well?

Reply:
We do not support multisig in deployer yet